index.js

```diff
@@ -0,0 +1,57 @@
+const omit = require('lodash/omit');
+const { Container, config, format, transports } = require('winston');
+
+const { combine, colorize, label: addLabel, printf } = format;
+
+const container = new Container();
+const TEST = process.env.NODE_ENV === 'test';
+
+let LEVEL = 'info';
+
+const appendMeta = format(i => {
+  const meta = omit(i, ['level', 'message', 'label']);
+  if (!Object.keys(meta).length) return i;
+  i.message = `${i.message}\t${JSON.stringify(meta, null, 2)}`; // eslint-disable-line
+  return i;
+});
+
+class ConsoleTransport extends transports.Console {
+  constructor(opts) {
+    super(opts);
+    this.silent = opts.silent;
+  }
+  log(...args) {
+    if (this.silent) return;
+    super.log(...args);
+  }
+}
+
+function add(id, label = id) {
+  container.add(id, {
+    level: LEVEL,
```

Comment:
wdyt of doing

```js
get level: () => LEVEL,
```

instead of having to loop through all containers below when changing level?

Reply:
i can try, not sure if it'll work, any Object.assign along the way and it'll kill the getters

Reply:
ok fair enough just trying to be cute, no big deal if it's not robust